core/src/main/res/layout/activity_bookmarks.xml

```diff
@@ -3,10 +3,31 @@
   xmlns:app="http://schemas.android.com/apk/res-auto"
   android:layout_width="match_parent"
   android:layout_height="match_parent"
+  xmlns:tools="http://schemas.android.com/tools"
   android:fitsSystemWindows="true">
 
-  <include layout="@layout/layout_standard_app_bar" />
+  <com.google.android.material.appbar.AppBarLayout
+    android:id="@+id/app_bar"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:layout_constraintTop_toTopOf="parent"
+    android:background="@color/black"
+    tools:showIn="@layout/activity_help">
+
+    <include layout="@layout/layout_toolbar" />
+
+    <Switch
```

Comment:
this switch and the bookmarks switch now share a lot of attributes, right click>refactor>extract style>whole project and if you get lucky it will hit the switch in historyActivity too. A quirk of extraction is that it doesn't take `app` attributes, you will have to add this manually to the style, omitting the `app` prefix